Test-NetStack.psm1

```diff
@@ -156,6 +156,17 @@ Function Test-NetStack {
     $Global:ProgressPreference = 'SilentlyContinue'
     Clear-Host
 
+    $LogFileParentPath = Split-Path -Path $LogPath -Parent -ErrorAction SilentlyContinue
+
+    if (-not (Test-Path $LogFileParentPath -ErrorAction SilentlyContinue)) {
+        $null = New-Item -Path $LogFileParentPath -ItemType Directory -Force -ErrorAction SilentlyContinue
+    }
+
+    $LogFile = New-Item -Path $LogPath -ItemType File -Force -ErrorAction SilentlyContinue
+
+    "Starting Test-NetStack - $([System.DateTime]::Now)" | Out-File $LogFile -Append -Encoding utf8 -Width 2000
+    "`r`n" | Out-File $LogFile -Append -Encoding utf8 -Width 2000
```

Comment:
"Save the processors"...Add the `r`n to the previous line. e.g.

::Now)`r`n" | out-file $logfile...